packages/elements/src/components/app/app.component.ts

```diff
@@ -136,6 +141,15 @@ export class MutationTestReportAppComponent extends LitElement {
   }
 
   private updateModel(report: MutationTestResult) {
+    if (!this.report) {
```

Comment:
Why does this method accept a `report` and at the same time uses `this.report`?

Reply:
I think this is a leftover, but this could probably be removed and use the `report` parameter instead.